src/mlpack/core/arma_extend/arma_extend.hpp

```diff
@@ -55,12 +52,6 @@ namespace arma {
   // u64/s64
   #include "hdf5_misc.hpp"
 
-  // ccov()
-  #include "op_ccov_proto.hpp"
-  #include "op_ccov_meat.hpp"
-  #include "glue_ccov_proto.hpp"
-  #include "glue_ccov_meat.hpp"
-  #include "fn_ccov.hpp"
```

Comment:
Agreed, I think dropping `ccov()` from `arma_extend` is the right thing to do here.